test_runner/src/main/kotlin/ftl/reports/CostReport.kt

```diff
@@ -41,5 +42,6 @@ object CostReport : IReport {
         val output = generate(matrices)
         if (printToStdout) print(output)
         write(matrices, output, args)
+        GcStorage.uploadReportResult(output, args, fileName())
```

Comment:
Just want to confirm, we want to upload all reports or just junit's?

Reply:
I think we want to upload all reports.